hubmap/query_app/filters.py

```diff
@@ -256,20 +260,15 @@ def get_cell_type_filter(query_params: Dict) -> Q:
     if input_type == "cell_type":
         return Q(grouping_name__in=input_set)
 
-    if input_type == "cell":
-
-        cell_qs = Cell.objects.filter(cell_id__in=input_set)
-
-        cell_type_pks = cell_qs.distinct("cell_type").values_list("cell_type", flat=True)
-
-        q = Q(pk__in=cell_type_pks)
-
-        return q
-
-    if input_type == "dataset":
-
-        cell_qs = Cell.objects.filter(dataset__uuid__in=input_set)
+    entities_dict = {
```

Comment:
I like this type of dispatch based on entity type -- much cleaner than the `if`/`elif`/... blocks.